src/extensions/addrbar/bootstrap-table-addrbar.js

```diff
@@ -63,6 +63,23 @@ function _buildUrl (dict, url = window.location.search) {
   return url
 }
 
+/*
+* function: _updateHistoryState
+* var _prefix = this.options.addrPrefix || ''
+* _updateHistoryState(_prefix)
+* returns void
+*/
+function _updateHistoryState ( _prefix ) {
+  const params = {}
+  params[`${_prefix}page`] = this.options.pageNumber,
```

Comment:
That cant work, because of `this`.

Reply:
Add `this` as parameter then it should work.
Its not the best way to solve it, but the extension must be rewrite anyways.

Reply:
It also would be greate to check if `order`, `sort` and/or `search` are `empty` or `undefined`.
If that is the case we should not add them as url parameter (`order=undefined` looks bad).

Reply:
I've change call of this function and removes 'undefined' from urls